mifospay/src/main/res/values/strings.xml

```diff
@@ -159,6 +159,7 @@
     <string name="disable_account">Disable account</string>
     <string name="log_out">Log out</string>
     <string name="debit_card_details">Debit Card Details</string>
+    <string name="debit_card_error_message">Invalid Debit Card</string>
```

Comment:
In my opinion, be more specific with the error.

Reply:
More specific?
like, Invalid Debit card number?

Reply:
Yes

Reply:
Done
Any other modifications needed?